### ISSUE TITLE
Controls remain on screen after switching to fullscreen before video has loaded

### ISSUE THREAD

**Issue by uusijani** [TeamNewPipe/NewPipe#4950]
### Checklist

- [x] I am using the latest version - v0.20.3
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo.
- [x] I have read the contribution guidelines
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
0. Have a somewhat slow-to-load video (for whatever reason)
1. Open the video in Newpipe
2. Before video details (such as duration) have loaded, tap to switch to fullscreen

### Actual behaviour
The playback controls remain on screen while the video plays, I have to tap once more to make them go away.

### Expected behavior
The playback controls should disappear like they do when switching to fullscreen _after_ the video details have loaded (and playback has begun).

### Screenshots/Screen recordings

![fullscreen](https://user-images.githubusercontent.com/4244772/99875991-7f27cd80-2bfc-11eb-83ae-be8ee59022ab.gif)

### Device info

 - Android version/Custom ROM version: Android 10 with OneUI 2.5
 - Device model: Samsung Galaxy Tab S5e

**Comment by TacoTheDank** [comment]
Hello, does this issue still occur on the latest version?

**Comment by TacoTheDank** [comment]
Interesting... I also have a Samsung device running Android 10 with OneUI 2.5, however mine is a Note 9 (a phone). I am also unable to reproduce this behavior. I think this may be an issue regarding the tablet UI.

Even if they don't automatically disappear, are they at least dismissible with a touch?

This may be fixable with the planned player rewrite. We'll just have to see.

**Comment by uusijani** [comment]
Yes, they're dismissible. They can even be brought back on after that (with another tap), and then they do disappear on their own.

I've updated Android on the tablet since reporting this, it's now running Android 11 with One UI 3.1, with no change in the behavior as reported.

I've also tested landscape, portrait and autorotate modes, all with the same result.

**Comment by Feuerswut** [comment]
Ah, yes, sometimes this still happens. Click twice and it should disappear normally
